macros/edr/system/system_utils/empty_table.sql

```diff
@@ -7,7 +7,7 @@
 {% endmacro %}
 
 {% macro empty_data_monitoring_metrics() %}
-    {{ elementary.empty_table([('id','string'),('full_table_name','string'),('column_name','string'),('metric_name','string'),('metric_value','float'),('source_value','string'),('bucket_start','timestamp'),('bucket_end','timestamp'),('bucket_duration_hours','int'),('updated_at','timestamp')]) }}
+    {{ elementary.empty_table([('id','string'),('full_table_name','string'),('column_name','string'),('dimension','string'),('dimension_value','string'),('metric_name','string'),('metric_value','float'),('source_value','string'),('bucket_start','timestamp'),('bucket_end','timestamp'),('bucket_duration_hours','int'),('updated_at','timestamp')]) }}
```

Comment:
I see here that the order of the columns is different than in the test queries

Reply:
You are right.
I tested it against existing table and it didn't matter due to the incremental.
Fixing :)